page_templates/style_guide.md

```diff
@@ -0,0 +1,73 @@
+
+# Jina AI Documentation Style Guide
+
+Guides are publicly available and represent the Jina AI brand and the quality of the product. Please adhere to the following steps to ensure basic professionalism in the docs.
+
+## Guiding principles:
+
+-   **Accessibility**. Everyone who needs to should understand our documents.
+
+-   **Professional**. A consistent writing and formatting style will give a professional look to the documents.
```

Comment:
Professionalism?

Maybe `documents` -> `documentation`? Since we already discuss Documents as part of Jina and there's room for misunderstanding

Reply:
Good point, Done